src/main/java/org/ibp/api/brapi/v1/germplasm/Germplasm.java

```diff
@@ -86,6 +86,14 @@
 	@JsonView(View.GermplasmBrapiV1_3.class)
 	private final static String documentationURL = null;
 
+	@JsonView(View.GermplasmBrapiV1_3.class)
```

Comment:
studyDbId and trialName seems to be attributes of a new class called GermplasmSummaryList that contains a list of germplasm @vmaletta

Reply:
@vmaletta please remove trialName and studyDbId.